libyul/backends/evm/StackLayoutGenerator.cpp

```diff
@@ -191,13 +256,11 @@ Stack createIdealLayout(Stack const& _operationOutput, Stack const& _post, Calla
 }
 }
 
-Stack StackLayoutGenerator::propagateStackThroughOperation(Stack _exitStack, CFG::Operation const& _operation)
+Stack StackLayoutGenerator::propagateStackThroughOperation(Stack _exitStack, CFG::Operation const& _operation, bool _aggressiveStackCompression)
```

Comment:
https://github.com/ethereum/solidity/pull/11824 would allow forcing
``_aggressiveStackCompression`` to ``true``, if ``_operation`` is a recursive function call, hence
lowering the overall stack pressure during the recursion.

I.e. being able to add
```
if (auto const* functionCall = get_if<CFG::FunctionCall>(&_operation.operation))
  if (functionCall->recursive)
    _aggressiveStackCompression = true;
```
is the purpose of https://github.com/ethereum/solidity/pull/11824

Reply:
Done this by now.